.pre-commit-config.yaml

```diff
@@ -33,3 +33,11 @@ repos:
     rev: v0.1.15a2
     hooks:
     - id: clean-notebook
+
+-   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
I'm -1 on adding type checking to pre-commit. A pre-commit run should be fast, and type checking is slow. And could even cause problems because of caching.

Reply:
OK. Will remove